README.md

```diff
@@ -52,33 +52,37 @@ Note: to run all examples with visualizations, install including extra requireme
 If you intend to use `htm.core` as a library that provides you Python \& C++ HTM, 
 you can use our [binary releases](https://github.com/htm-community/htm.core/releases).
 
-
-### Building from Source
-
-Fork or download the HTM-Community htm.core repository from https://github.com/htm-community/htm.core
-
 #### Prerequisites
 
-- [CMake](http://www.cmake.org/)  Version 3.8  (3.14 for Visual Studio 2019)
+For running C++ apps/examples/tests from binary release: none. 
+If you want to use python, then obviously:
+
 - [Python](https://python.org/downloads/)
     - Version 3.4+ (Recommended)
     - Version 2.7
       + We recommend the latest version of 2.7 where possible, but the system version should be fine.
       + Python 2 is Not Supported on Windows, use Python 3 instead.
-      + Python 2 is not tested by our CI anomore. It may still work but we don't test it. We expect to drop support for Python2 around 2020. 
+      + Python 2 is not tested by our CI anomore. It may still work but we don't test it. We expect to drop support for Python2 around 2020.
 
   Be sure that your Python executable is in the Path environment variable.
   The Python that is in your default path is the one that will determine which
   version of Python the extension library will be built for.
   - NOTE: People have reported success with `Anaconda` python.
   - Other implementations of Python may not work.
   - Only the standard python from python.org have been tested.
-- Python tools: In a command prompt execute the following.
-```
-cd to-repository-root
-python -m pip install --user --upgrade pip setuptools setuptools-scm wheel
-python -m pip install --no-cache-dir --user -r requirements.txt
-```
+
+
+### Building from Source
+
+Fork or download the HTM-Community htm.core repository from https://github.com/htm-community/htm.core
+
+#### Prerequisites
+
+- same as for Binary releases, plus:
+- [CMake](http://www.cmake.org/)  Version 3.8  (3.14 for Visual Studio 2019)
```

Comment:
Actually it is version **3.7** and 3.14 for MSVC 2019
but probably close enough.

Reply:
Thanks, fixed! 
Thank you for review, merging now to avoid useless CI checks.